instrumentation/runtime-metrics/runtime-metrics-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetricsjava17/HandlerRegistry.java

```diff
@@ -38,7 +38,7 @@
 import javax.annotation.Nullable;
 
 final class HandlerRegistry {
-  private static final String SCOPE_NAME = "io.opentelemetry.instrumentation.runtimetelemetryjfr";
+  private static final String SCOPE_NAME = "io.opentelemetry.instrumentation.runtime-metrics";
```

Comment:
Same comment as in the other place:
```suggestion
  private static final String SCOPE_NAME = "io.opentelemetry.runtime-metrics-java17";
```

Reply:
updated